### DISCUSSION TITLE
Python has stopped working

### DISCUSSION THREAD

**Discussion by gwolf01** [comfyanonymous/ComfyUI#4008] [Q&A]
Every time I go to load/run comfy U,I in the middle of the process, I will get a pop up that says "Python has stopped working"; and then I can close that pop up and the load will continue to run until the pop up comes up again; and then sometimes it'll continue to open the program but mostly other times it'll stop running and I'll have to try again. I have tried reloading python and Python embedded to no avail I have updated comfy ui to its latest version I am at a loss! can I please get help?!

**Comment by ltdrdata** [comment]
Based on the symptoms, it appears that part of the installation files may be corrupted, or it could be a hardware or driver issue.
Have you tried testing with the portable version installed and in its default state without installing any custom nodes?

**Reply by gwolf01** [reply]
Yes, I've checked installation files and for corrupted files as much as my knowledge allows; I have multiple comfy apps on various drives and they are all doing the same thing (on the same desktop) on startup and sometimes after it has been loaded for a while. I've checked for memory issues, hardware issue, reinstalled python, I just don't know what I'm missing.

**Reply by ltdrdata** [reply]
> Yes, I've checked installation files and for corrupted files as much as my knowledge allows; I have multiple comfy apps on various drives and they are all doing the same thing (on the same desktop) on startup and sometimes after it has been loaded for a while. I've checked for memory issues, hardware issue, reinstalled python, I just don't know what I'm missing.

How did you install ComfyUI? Did you use the portable version?

**Reply by gwolf01** [reply]
I downloaded the ZIP portable version and went from there; all have been working great until this past week and I have not changed anything except to update comfyui.

**Reply by ltdrdata** [reply]
In that case, try using Python venv for configuration instead of portable.
